### DISCUSSION TITLE
When can we expect a stable release?

### DISCUSSION THREAD

**Discussion by meenbeese** [lichess-org/mobile#171] [Q&A] [answered]
Hello Mr. @veloce

I have been closely following the development of the app and am excited to finally use the new app to replace the old one. I don't want to bug you with questions, but can you estimate a release date or timeline for the new app?

Alternatively, I heard of some testers trying out an early version on Discord. But I did find any links to that Discord server on Reddit or anywhere else. I would also appreciate being able to try the app early while waiting for a stable release.

Thank you.

**Comment by vaibhavsection** [comment]
@veloce 
Please reply , approximate realise date of new app

**Comment by akagr** [comment, marked as answer]
@vaibhavsection @meenbeese Hey guys I know you're as excited as me about the new app. Friendly reminder that it's not considered good etiquette to open Github issues asking for release date on something we don't pay for.

That said, I just built the app and ran it on my phone. I can play the puzzles etc, but I don't see an option to play against other players yet. Considering how core that feature is to Lichess, it not being in master yet means you shouldn't hold your breath for beta tests starting any time soon.

PS: I didn't mean to offend. I totally share your enthusiasm for Lichess it is awesome to follow the development. You'll never know when sign ups for beta tests open up. I hope it's awesome, whenever the authors consider it stable 🤞 😄.

**Comment by veloce** [comment]
No release date at this moment indeed.

**Reply by vaibhavsection** [reply]
But try to release early access (beta version) as soon as possible.

**Reply by M7rc7ry** [reply]
I guess people are keen to use it to play a challenge most of all it seems, sounds like its going to be a hit.
